modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2873,6 +2874,19 @@
 	end
 
 
+	function m.scanSourceForModuleDependencies(cfg)
+		if _ACTION >= "vs2019" then
+			if cfg.scanformoduledependencies ~= nil then
```

Comment:
So if the value isn't specified in the premake project file, we just use the IDE default value?

Reply:
Yes.  If the option isn't used then nothing gets written to the project file and the IDE's default is used.